### ISSUE TITLE
Is this project alive?

### ISSUE THREAD

**Issue by usamec** [AtheMathmo/rusty-machine#181]
I haven't seen update in 6 months and they PR seem without response.

**Comment by AtheMathmo** [comment]
I'm still holding out hope that I will be able to pick things up in this project again soon. Unfortunately, for now my schedule is a little too intense for me to dedicate any serious time.

In the mean time I am happy to discuss providing the necessary permissions to anyone who would like to maintain the project (even just temporarily).

TLDR; it's still breathing but low on stamina.

**Comment by atvaccaro** [comment]
I'm new to Rust but interested in helping contribute; I took undergrad ML so I have a rough understanding of how everything works, if not the details (yet).

**Comment by NivenT** [comment]
@AtheMathmo if you are still open to allowing someone else to temporarily maintain this project, and are waiting for the right person, I'd suggest @zackmdavis on the relatively simple condition that he's had the most contributions, and so is probably one of the people most familiar with the code base. This is of course assuming he's ok with it. I haven't asked him or anything; this is just a (possibly unwanted) suggestion.

**Comment by zackmdavis** [comment]
@NivenT (Thanks for the vote of support; I certainly wouldn't refuse a commit bit if offered, but be it noted that I also have a lot of non-code life priorities right now)

**Comment by zackmdavis** [comment]
@AtheMathmo gave me commit privs; been a bit busy myself, and I'm offline tomorrow, but I hereby pledge to spend 1 hour on Sunday the twenty-second reviewing the pull-request backlog

**Comment by zackmdavis** [comment]
Yes.

**Comment by zackmdavis** [comment]
@atvaccaro the more the merrier!

**Comment by XX** [comment]
@zackmdavis @AtheMathmo Is this project currently maintained?

**Comment by zackmdavis** [comment]
@XX thanks for reaching out! I'm worried that the _realistic_ answer is "No". Original author @AtheMathmo is busy with grad school; I graciously accepted a commit bit as was suggested above, optimistically thinking that I would be able to spare the time ... and in my defense, I _have_ merged a few pull requests!—but I haven't actually made it a priority, and we still have open PRs that would require serious reviewer effort that I haven't put in. Most of my open-source time lately has been on [rustc](https://github.com/rust-lang/rust/commits?author=zackmdavis) and I don't see that changing soon ... we may want to put together a plan to either (a) find a dedicated maintainer, or (b) find a day or two to close out/merge up the remaining pull requests and edit the README to clarify the currently-unmaintained status.

@AtheMathmo thoughts?

**Comment by AtheMathmo** [comment]
I agree with your evaluation. I don't have time to support any serious development directly. I would be happy with either option (a) or (b) going forwards.

If somebody wants to take on ownership I would be happy to talk with you about the project and what that might entail. Otherwise, lets try to find some time to tidy up existing PRs (I don't want the effort of others to be wasted) and then we can mark the library as unmaintained and hope that someone may pick it up later!
